JavaDirectory/src/main/java/com/bug/free/invention/api/controllers/US002.java

```diff
@@ -0,0 +1,92 @@
+package com.bug.free.invention.api.controllers;
+
+import com.bug.free.invention.api.classes.Job;
+import org.json.JSONObject;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+    @RequestMapping("/api/jobs")
+    public class US002 {
+
+        @GetMapping("/JSONDemo")
+        public List<Employee> demoJson(){
+            List<Employee> returnList = new ArrayList<Employee>();
+            returnList.add(new Employee(1,"name","difjidsfj"));
+            returnList.add(new Employee(2,"ijisdfjdsif","dsuifjhduisfhui"));
+            return returnList;
+        };
+
+/*    @GetMapping("/jobCapability")
+    public String  getJobCapability(){
+        System.out.println("Attempting connection");
+        try{
+            Statement statement = DBConfig.getConnection().createStatement();
+            String dbQuery = "SELECT Job.Job_Title, Capability.capabilityName FROM Job JOIN Capability ON(Job.capabilityID = Capability.capabilityID);";
+            ResultSet results = statement.executeQuery(dbQuery);
+            JSONObject return_json= new JSONObject();
+            int i = 0;
+            while(results.next()) {
+                String job_title = results.getString("Job_Title");
+                String capabilityname = results.getString("capabilityName");
+                JSONObject tempJson = new JSONObject("{\"job_title\": \"" + job_title + "\", \"capabilityname\": \""+ capabilityname+ "\"}");
+                return_json.put(String.valueOf(i), tempJson);
+                i++;
+            }
+            return return_json.toString();
+        }
+        catch (Exception e){
+            e.printStackTrace();
+            return null;
+        }
+    }*/
+
+        @GetMapping("/jobRoles") //US001
+        public List<Job> getJobRoles(){
+            System.out.println("Attempting connection");
+            try{
+
+                Statement statement = DBConfig.getConnection().createStatement();
+                String dbQuery = "SELECT job.jobID, job.jobTitle, job.bandID, capability.capabilityID, capability.capabilityName, band.bandName, band.bandLevel, jobFamily.jobFamilyTitle FROM job JOIN capability ON(job.capabilityID = capability.capabilityID) JOIN band ON(job.bandID = band.bandID) JOIN jobFamily ON(job.jobFamilyID = jobFamily.jobFamilyID);";
+                ResultSet results = statement.executeQuery(dbQuery);
+                List<Job> Jobs = new ArrayList<Job>();
```

Comment:
Variables should not be Capitalised, and you don't need to repeat the generic type.
```suggestion
                List<Job> jobs = new ArrayList<>();
```